dev/tests/integration/testsuite/Magento/Customer/Block/Adminhtml/Edit/Tab/NewsletterTest.php

```diff
@@ -59,6 +59,8 @@ protected function tearDown(): void
 
     /**
      * @magentoDataFixture Magento/Customer/_files/customer_sample.php
+     * @magentoDataFixture Magento/Newsletter/_files/newsletter_sample.php
```

Comment:
I would recommend make new test case for that and leave old as is

Reply:
done